node/src/database/db_initializer.rs

```diff
@@ -86,16 +82,22 @@ impl DbInitializer for DbInitializerReal {
         match Connection::open_with_flags(database_file_path, flags) {
             Ok(conn) => {
                 eprintln!("Opened existing database at {:?}", database_file_path);
-                Self::conn_special_setup(&conn, &init_config)?;
+                Self::special_conn_configuration(&conn, &init_config)?;
                 let config = self.extract_configurations(&conn);
+                let found_schema_version = config.get("schema_version");
```

Comment:
I'm advising to hide the following match statement in a separate function. It takes too much space counter the length and focus of this big function.